src/states_screens/race_gui.cpp

```diff
@@ -390,6 +404,65 @@ void RaceGUI::drawGlobalTimer()
 
 }   // drawGlobalTimer
 
+
+//-----------------------------------------------------------------------------
+/** Displays the live difference with a ghost on screen.
+ */
+void RaceGUI::drawLiveDifference()
+{
+    assert(World::getWorld() != NULL);
+
+    if (!World::getWorld()->shouldDrawTimer())
+    {
+        return;
+    }
+
+    const LinearWorld *linearworld = dynamic_cast<LinearWorld*>(World::getWorld());
+    assert(linearworld != NULL);
+
+    // Don't display the live difference timer if its time is wrong
+    // (before crossing the start line at start or after crossing it at end)
+    if (!linearworld->hasValidTimeDifference())
+        return;
+
+    float live_difference = linearworld->getLiveTimeDifference();
+
+    int timer_width = m_small_precise_timer_width;
+    if (fabsf(live_difference) >= 59.9995f)
```

Comment:
What is this magic number?

Reply:
There are 60s to a minute (I think we can assume that this won't change anytime soon).

Now, add that it is rounded to milliseconds. What's the smallest number of seconds who'd get rounded to 1 minutes ? This is it.

Reply:
implicit in my question was : adding a comment like "// smallest number of seconds that could get rounded to 1 minute" would help clarify a lot. Or even better, if this is about rounding, then checking "rounding(...) >= 60; // 60 seconds" would be even clearer

Reply:
Using a rounding function is not very practical - I'd need to multiply by 1000 to round to milliseconds, then divide by 1000 again to compare to 60. I've added a comment.